dune_api_scripts/update_appdata_view.py

```diff
@@ -7,7 +7,7 @@
 from duneapi.types import DuneQuery, Network, QueryParameter
 from duneapi.util import open_query
 
-from .utils import app_data_entries
+from dune_api_scripts.utils import app_data_entries
```

Comment:
Nice! Good catch here.

Reply:
That was actually supposed to be a part of the suggestions PR, but thanks!